src/components/Reports/Reports.jsx

```diff
@@ -1,17 +1,223 @@
 import React, { Component } from 'react'
+import { fetchAllProjects } from '../../actions/projects'
+import { connect } from 'react-redux'
+import { getAllUserTeams } from '../../actions/allTeamsAction';
+import TeamTable from './TeamTable'
+import PeopleTable from './PeopleTable'
+import ProjectTable from './ProjectTable'
+import TasksTable from './TasksTable'
+import { getAllUserProfile } from '../../actions/userManagement';
+import { fetchAllTasks } from "../../actions/task";
+import moment from 'moment'
+
 
 class ReportsPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showProjects: false,
+      showPeople: false,
+      showTeams: false,
+      teamNameSearchText: '',
+      teamMembersPopupOpen: false,
+      deleteTeamPopupOpen: false,
+      createNewTeamPopupOpen: false,
+      teamStatusPopupOpen: false,
+      wildCardSearchText: '',
+      selectedTeamId: 0,
+      selectedTeam: '',
+      checkActive: '',
+      formElements: {
+        summary: '',
+        summaryLastWeek: '',
+        summaryBeforeLast: '',
+        mediaUrl: '',
+        weeklySummariesCount: 0,
+        mediaConfirm: false
+      },
+      dueDate: moment().tz('America/Los_Angeles').endOf('week').toISOString(),
+      dueDateLastWeek: moment().tz('America/Los_Angeles').endOf('week').subtract(1, 'week').toISOString(),
+      dueDateBeforeLast: moment().tz('America/Los_Angeles').endOf('week').subtract(2, 'week').toISOString(),
+      activeTab: '1',
+      errors: {},
+      fetchError: null,
+      loading: true,
+    }
+    this.showProjectTable = this.showProjectTable.bind(this);
+    this.showPeopleTable =this.showPeopleTable.bind(this);
+    this.showTeamsTable=this.showTeamsTable.bind(this)
+    this.setActive=this.setActive.bind(this)
+    this.setInActive=this.setInActive.bind(this)
+    this.setAll=this.setAll.bind(this)
+  }
+
+  componentDidMount() {
+    this.props.fetchAllProjects(); // Fetch to get all projects
+    this.props.getAllUserTeams();
+    this.state = {
+      showProjects: false,
```

Comment:
I think you can set one of these to true as the 'default' value that will be shown.